### DISCUSSION TITLE
Admin templates with fields?

### DISCUSSION THREAD

**Discussion by tropcicstefan** [OrchardCMS/OrchardCore#9071] [Q&A] [answered]
I am having trouble making admin templates on part or type for fields through code or UI.
 For instance what is the template name for 'Subtitle' text field(standard editor) on 'ArticlePart' content part in 'Article' content type.
Tried:
ArticlePart_Edit__Subtitle
Article_Edit__ArticlePart__Subtitle

Is it even possible or are editors alternates/templates?
 
I know I hooked theme correctly because I can alter parts on types. For instance Article__FlowPart.Edit.cshtml is working correctly.

**Comment by ns8482e** [comment, marked as answer]
Editor for Field is driven by Editor Options. Create option shape `{FIELDTYPE}_Option__{EDITORNAME}` and editor shape `{FIELDTYPE}_Edit__{EDITORNAME}` to customize field editor.

[Documented here](https://docs.orchardcore.net/en/dev/docs/reference/modules/ContentFields/index.html#creating-custom-editors)

**Reply by tropcicstefan** [reply]
I thought so, thx.  Just wasn't sure if there is something similar to part alternative
